platform/security/security-services-app/src/main/resources/features.xml

```diff
@@ -61,6 +61,8 @@
         <feature prerequisite="true">security-expansion</feature>
         <feature prerequisite="true">security-expansion-user-attributes</feature>
         <feature prerequisite="true">security-expansion-metacard-attributes</feature>
+        <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.commons-beanutils/1.8.3_2</bundle>
```

Comment:
Is it absolutely necessary to include this bundle? There is a CVE against commons-beanutils and I'd rather not include it here.